client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -313,6 +315,14 @@ export const SeoForm = React.createClass( {
 			updatedOptions.advanced_seo_front_page_description = this.state.frontPageMetaDescription;
 		}
 
+		// Empty arrays for title formats were not working properly with saveSiteSettings.
```

Comment:
we probably don't need to mention that this code was broken in the past, only explain why it's here now

> Since the absence of data indicates that there are no changes in the network request
> we need to send an indicator that we specifically want to clear the format
> We will pass an empty string in this case